NEWS.md

```diff
@@ -1,5 +1,46 @@
 # OSBuild Composer - Operating System Image Composition Services
 
+## CHANGES WITH 13:
+
+  * Fedora IoT is now supported for Fedora 32 in the form of producing the
+    commit tarball. Feel free to test it and report any issues you find.
+
+  * All RHEL 8 minor releases are now merged into one. This means
+    osbuild-composer will always build up-to-date RHEL 8 images.
+
+  * The timeout for AWS upload is removed. It's very hard to predict how long
+    will the AWS upload take. With the timeout in place, it caused the test
+    suite to produce a lot of false positives.
+  
+  * Build logs were broken in the previous release, this release fixes it.
+    This time, they were properly saved but weldr API read them from a wrong
+    location. This is now fixed and covered with basic tests.
+    
+  * The weldr API socket has now correct permissions. As the result, it can
+    be read and written only by root and the weldr group. This is the same
+    behaviour as Lorax has.
+    
+  * By mistake, workers incorrectly used the default store for every build.
+    However, the caching is still not tested properly, so this release
```

Comment:
The issue is not with testing, just that the cache would grow too large as we still commit the images to it.

Reply:
Thanks, will change this!